### PR TITLE
THREESCALE-10848 remove toc generator

### PR DESCRIPTION
**Pull request by MStokluska**

### CODE REVIEW

File: make/toc.mk [hunk 1]
```diff
@@ -1,19 +0,0 @@
-
```
Comment (austincunningham) [inline make/toc.mk:1]: presume this is removing the make directory

Comment (MStokluska) [reply make/toc.mk:1]: yea, entire file got removed

**Review by austincunningham — APPROVED** [review]

**Comment by austincunningham** [comment]
won't miss this test

**Pull request merged**